client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -160,30 +159,39 @@ var CheckoutThankYou = React.createClass( {
 
 			if ( purchases.some( isJetpackPremium ) ) {
 				componentClass = JetpackPremiumPlanDetails;
+				primaryPurchase = find( purchases, isJetpackPremium );
```

Comment:
I added e1fc410 to handle this. @drewblaisdell feel free to remove it if you think this added unnecessary complexity.